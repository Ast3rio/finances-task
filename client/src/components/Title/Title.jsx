import React from 'react';
import styles from './Title.module.scss';

export const Title = ({title}) => {
    return <h2 className={styles.title}>{title}</h2>
};

