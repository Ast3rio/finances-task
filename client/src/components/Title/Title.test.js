import React from "react";
import {Title} from "./Title";
import renderer from 'react-test-renderer';

describe('Test <Title/>', () => {
    it('Title have renderer correctly', () => {
        const title = renderer.create(<Title />).toJSON();
        expect(title).toMatchSnapshot();
    })
})

