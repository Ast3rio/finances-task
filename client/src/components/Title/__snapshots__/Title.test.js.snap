// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Title/> Title have renderer correctly 1`] = `
<h2
  className="title"
/>
`;
